client/telemetry/src/layer.rs

```diff
@@ -66,29 +66,15 @@ where
 							json: Some(json),
 							..
 						} => {
-							eprintln!(
-								//target: "telemetry",
-								"###### sent message as {}: {}", id, json,
-							);
-							if let Err(err) = sender.try_send((
+							let _ = sender.try_send((
 								message_verbosity
 									.try_into()
 									.expect("telemetry log message verbosity are u8; qed"),
 								json.clone(),
-							)) {
-								// TODO logs dont work here
-								log::warn!(
-									target: "telemetry",
-									"Ignored telemetry message because of error on channel: {:?}",
-									err,
-								);
-							}
+							));
 						}
 						_ => panic!("missing fields in telemetry log: {:?}", event),
```

Comment:
Same than here: https://github.com/paritytech/substrate/pull/7463#discussion_r526716808

It never happens but it could technically happen if the user decide to not use the macro `telemetry!` and mess up with the fields.

Reply:
Done in db42f20c6